src/components/vertical-menu/responsive-vertical-menu/responsive-vertical-menu.component.tsx

```diff
@@ -115,10 +117,10 @@ const BaseMenu = ({
   );
 
   const handleActiveToggle = useCallback(() => {
-    setActive((previousState) => !previousState);
+    setActive(!active);
```

Comment:
question: I think it's better to use a callback to avoid a stale value being negated as state is set async, was there a reason this was changed?